lib/PuppeteerSharp/NetworkManager.cs

```diff
@@ -180,6 +183,7 @@ private void OnResponseReceived(MessageEventArgs e)
                     request,
                     (HttpStatusCode)e.MessageData.response.status,
                     ((JObject)e.MessageData.response.headers).ToObject<Dictionary<string, object>>(),
+                    (bool)e.MessageData.response?.fromDiskCache,
                     e.MessageData.response.securityDetails?.ToObject<SecurityDetails>());
```

Comment:
Let's create a `MessageResponse` for the `Response` object so we start working also on #97